src/utils/http/fetchAPI.js

```diff
@@ -34,6 +36,11 @@ const fetchAPI = async (path, params = {}, extras = {}) => {
       // just return the error code, this happens in many tests
       // where we mock a string response instead of proper json
     }
+
+    if (response.status === 503 && data?.message === 'Maintenance mode') {
+      pushNotificationMessage('error', 'The system is undergoing scheduled maintenance, the platform may not be accessible for a few hours.');
+    }
```

Comment:
I would add a return after the `pushNotification` because otherwise it shows that the system is in maintenance but still shows more errors that are not relevant

Reply:
I can comment out the `pushNotificationMessage`'s in `handleError` otherwise, that would catch some other errors.

Let me know